cnd/src/swap_protocols/rfc003/create_swap.rs

```diff
@@ -56,16 +56,17 @@ pub async fn create_swap<D, A: ActorState>(
         match generator.async_resume().await {
             // every event that is yielded is passed on
             GeneratorState::Yielded(event) => {
+                log::info!("swap {} yielded event {}", id, event);
                 dependencies.update::<A>(&id, event);
             }
             // the generator stopped executing, this means there are no more events that can be
             // watched.
             GeneratorState::Complete(Ok(_)) => {
-                log::info!("Swap {} finished", id);
+                log::info!("swap {} finished", id);
```

Comment:
Oh, I've been converting everything I touch to upercase first character, @luckysori and I discussed and didn't reach consensus.  We should get team consensus to save churn.  I don' care either way, as long as we do it just one way.

Reply:
I prefer lowercase but I think that is a topic we shouldn't spend too much time on :sweat_smile: